lua/compe/completion.lua

```diff
@@ -199,7 +220,7 @@ Completion._display = function(context)
             -- Fix start_offset gap.
             local gap = string.sub(context.before_line, start_offset, source_start_offset - 1)
             for _, item in ipairs(source_items) do
-              if items_uniq[item.original_word] == nil or item.dup ~= true then
+              if items_uniq[item.original_word] == nil or item.original_dup ~= true then
```

Comment:
Probably, This will solve https://github.com/hrsh7th/nvim-compe/issues/42#issuecomment-735403487